src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -1346,7 +1346,7 @@ bool SystemDictionaryShared::should_be_excluded(InstanceKlass* k) {
   }
 
   if (k->is_hidden() && !is_registered_lambda_proxy_class(k)) {
-    warn_excluded(k, "Hidden class");
+    log_debug(cds)("Skipping %s: %s", k->name()->as_C_string(), "Hidden class");
```

Comment:
A ResourceMark is needed here for calling k->name()->as_C_string().

Reply:
Thanks, yes here needs a ResourceMark.